plugins/rikki/heroeslounge/classes/hotslogs/IDFetcher.php

```diff
@@ -11,8 +11,12 @@ public static function fetchIDs()
         $sloths = SlothModel::all();
 
         foreach ($sloths as $sloth) {
-            set_time_limit(30);
-            IDFetcher::fetchIDHeroesProfile($sloth);
+            set_time_limit(60);
+            $throttleTime = IDFetcher::fetchIDHeroesProfile($sloth);
+
+            if ($throttleTime > 0) {
+                sleep($throttleTime);
+            }
         }
     }
```

Comment:
This function is probably not actually needed anymore

Reply:
Will update this in a different PR for #154